### PR TITLE
Greek translation

### PR DESCRIPTION
**Pull request by veqtrus**
FAQs untranslated for now.

~~When I'm running the local website I don't get the option to switch to Greek (tested by modifying `en.json`). Not sure if there's something more I should do.~~ Solved!

### CODE REVIEW

File: web/locales/gr.json [hunk 1]
```diff
@@ -0,0 +1,1788 @@
+{
+    "panel-initial-text": {
+        "thisproject": "Αυτό το έργο είναι",
+        "opensource": "ανοικτού κώδικα",
+        "see": "δείτε τις",
+        "datasources": "πηγές δεδομένων",
+        "contribute": "Συνεισφέρετε <a href=\"%s\" target=\"_blank\">προσθέτοντας την περιοχή σας</a>"
+    },
+    "mobile-main-menu": {
+        "map": "electricityMap",
```
Comment (veqtrus) [inline web/locales/gr.json:10]: Should I translate this as "Map"?

**Comment by veqtrus** [comment]
The README for localisation suggests using ISO 3166-1 Alpha-2 codes but wouldn't ISO 639-1 codes be more accurate? In that case Greek would be `el`.

**Comment by veqtrus** [comment]
> When I'm running the local website I don't get the option to switch to Greek (tested by modifying `en.json`). Not sure if there's something more I should do.

I see I will need to modify [locales-config.json](https://github.com/tmrowco/electricitymap-contrib/blob/master/web/locales-config.json) as well. Should probably be added to the README.

**Comment by Kongkille** [comment]
Hey @veqtrus what's the status of this? I know we've talked a bit about the point of using ISO 639-1 language codes instead, but does that constrain the current PR?

**Comment by veqtrus** [comment]
@Kongkille it can be merged as is and I can make a new PR when I get to translating the rest.

**Review by FelixDQ — APPROVED** [review]
Thank you @veqtrus! :-)

**Pull request merged**